src/einsteinpy/coordinates/core.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+import astropy.units as u
+
+class Cartesian:
+    """
+    Class for Cartesian Coordinates and related transformations.
+    """
+
+    @u.quantity_input(x=u.km, y=u.km, z=u.km)
```

Comment:
Keep things in `u.m` by default. We follow SI units

Reply:
We are dealing with orbits, no-one would ever give parameters in SI units. km is a convenient unit.